packages/server/routes/api/events.js

```diff
@@ -281,6 +281,45 @@ router.get(
   }
 );
 
+router.put(
+  "/:eventId/status",
+  AuthController.authenticate,
+  eventIdValidator,
+  async (request, response) => {
+    const result = validationResult(request);
+
+    if (!result.isEmpty()) {
+      response.status(400).json({
+        status: "fail",
+        data: result.array(),
+      });
+      return;
+    }
+
+    const { userId, status } = request.body;
+    const { eventId } = request.params;
```

Comment:
Remember to use `matchedData(request)` to access the validated input data now instead of accessing `request.body` or `request.params` because we're using express-validator.

See [here](https://express-validator.github.io/docs/guides/getting-started#accessing-validated-data).